docs/wiki/framework/arsenal-framework.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: wiki
+title: ACE Arsenal Framework
+description: Explains how to interact with the ACE Arsenal API.
+group: framework
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 12
+  patch: 0
+---
+
+<div class="panel callout">
+    <h5>BI functions and variables from BI Virtual Arsenal DO NOT apply to ACE Arsenal!</h5>
+</div>
+
+<div class="panel callout">
+    <h5>Note:</h5>
+    <p>Units and objects that can have ACE Arsenal added to them will be called "Boxes" in this documentation.</p>
+</div>
+
+To quickly add a full ACE Arsenal to a box for all clients (do not use in init fields):  
+`[_box, true] call ace_arsenal_fnc_initBox`  
+`_box` being the object you wish to add ACE Arsenal to.
```

Comment:
make the whole thing a code block (three backticks) and add this as a comment after the line with initBox.
You could also add another comment saying something like

> or `this` when called from the box's init field

Reply:
is this what you're suggesting?:

```cpp
[_box, true] call ace_arsenal_fnc_initBox;
// To quickly add a full ACE Arsenal to a box for all clients
```

`_box` being the object you wish to add ACE Arsenal to. (or `this` when called from the box's init field)